tests/apps/ideas/migrations/0008_alter_idea_description.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.16 on 2025-01-06 14:07
+
+from django.db import migrations
+import django_ckeditor_5.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
@m4ra tbh I'm not entirely sure why it created this one just now

Reply:
Because the 0006 has textfield, while the idea model has ckeditor 5. You don't need to change the 0006.